<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLineEdit" name="samFIleLineEdit"/>
    </item>
    <item row="0" column="3" colspan="2">
     <widget class="QPushButton" name="choseFileButton">
      <property name="text">
       <string>Choose File</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5" colspan="2">
     <widget class="QPushButton" name="loadFileButton">
      <property name="text">
       <string>Load File</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7" colspan="2">
     <widget class="QPushButton" name="showLenButton">
      <property name="text">
       <string>Show Length</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="9">
     <widget class="QTreeWidget" name="chrLenList">
      <property name="columnCount">
       <number>2</number>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">2</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="showLinesEdit">
      <property name="text">
       <string>20</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="multiBox">
      <property name="text">
       <string>multiMap</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QCheckBox" name="gappedBox">
      <property name="text">
       <string>gapped</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QCheckBox" name="reverseBox">
      <property name="text">
       <string>reverse</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4" colspan="2">
     <widget class="QPushButton" name="fewReadsButton">
      <property name="text">
       <string>Few Reads</string>
      </property>
     </widget>
    </item>
    <item row="2" column="6" colspan="2">
     <widget class="QPushButton" name="rpkmButton">
      <property name="text">
       <string>calcRPKM</string>
      </property>
     </widget>
    </item>
    <item row="2" column="8">
     <widget class="QPushButton" name="bdButton">
      <property name="text">
       <string>calcBD</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
